### ISSUE TITLE
Speaker notes won't show slides if presentation url contains question mark

### ISSUE THREAD

**Issue by lizatop** [hakimel/reveal.js#1392]
When I have a reveal.js presentation opened with a ? in the URL, I can not get the slides in the speaker notes loaded, because the iframes url's params are appended with a ? and browsers do not like two question marks in an URL.

Suggested fix in /plugin/notes/notes.html.

change :
```javascript
var currentURL = data.url + '?' + params + '&postMessageEvents=true' + hash;
var upcomingURL = data.url + '?' + params + '&controls=false' + hash;
```
to :
```javascript
var urlSep = (data.url.indexOf('?') > 0) ? '&' : '?';
var currentURL = data.url + urlSep + params + '&postMessageEvents=true' + hash;
var upcomingURL = data.url + urlSep + params + '&controls=false' + hash;
```

**Comment by FranceBaril** [comment]
Using 3.2.0, I can open the slides in the speaker notes when path has ? in Safari, but the parameters are not passed/applied properly.

**Comment by demoneaux** [comment]
Closed by #1522.
